en_US/global/gadgets.md

```diff
@@ -0,0 +1,9 @@
+# Available Gadgets
+
+### Handheld Terminal
+
+A portable, wireless screen to access the network and all the items stored within. Requires an APN and power to operate (see the Wireless section of [Modules](/modules)).
```

Comment:
"an APN" should probably be "a wireless connection", since APN just means Access Point Name. You don't say a phone needs to be "on an SSID" to mean it needs to be connected to wifi.